### PR TITLE
Improve modal size

### PR DESCRIPTION
**Pull request by viniciusarre**
Chore: change min-height to date type container

### CODE REVIEW

**Comment by viniciusarre** [comment]
@bryan-gilbert, I am not sure if this is how you've intended me to work on this. But, in short, I have added a parent container to the `date-picker`, with a fixed `min-height`. Please, let me know your thoughts on it or if you have any alternatives on approaching this!

**Comment by bryan-gilbert** [comment]
This looks like a promising idea. Please send me a screen shot via slack.

File: client/src/inside/components/page/EhrElementForm.vue [hunk 1]
```diff
@@ -54,6 +55,12 @@
 
 </template>
 
+<style scoped>
+.date-container {
+    min-height: 29vh;
+  }
+</style>
+
```
Comment (bryan-gilbert) [inline client/src/inside/components/page/EhrElementForm.vue]: please revert all changes in this file.  I've tried to make this work too and can't see a clean solution.  As is, without these changed, the date picker can be accessed by scrolling the dialog body.  Good enough for a prototype.

File: client/src/app/components/AppDialogShell.vue [hunk 2]
```diff
@@ -181,7 +181,6 @@ export default {
 
 .dialog-body {
   overflow-y: auto;
-  min-height: 40vh;
```
Comment (bryan-gilbert) [inline client/src/app/components/AppDialogShell.vue]: yes please keep this removal.  We don't need it.

**Comment by bryan-gilbert** [comment]
Please rebase onto master, squash and resubmit.  Thanks

**Comment by bryan-gilbert** [comment]
This work has been transferred to PR #559

**Pull request closed without merging**